src/main/resources/avro/bdg.avdl

```diff
@@ -87,6 +87,47 @@ record RecordGroupMetadata {
   union { null, int } predictedMedianInsertSize = null;
   union { null, string } platform = null;
   union { null, string } platformUnit = null;
+
+  /**
+   The IDs of the processing steps that have been applied to this record group/
+   */
+  array<string> processingStepIds = [];
```

Comment:
Yeah, I was on the hedge about this and could've gone either way. Since you prefer it this way, I'll change it to `array<ProcessingStep>`.